lte/gateway/python/integ_tests/defs.mk

```diff
@@ -278,7 +278,30 @@ s1aptests/test_restore_config_after_non_sanity.py
 
 # Add the s1aptester integration tests with federation gateway
 FEDERATED_TESTS = federated_tests/s1aptests/test_attach_detach.py \
-federated_tests/s1aptests/test_attach_detach_multi_ue.py
+federated_tests/s1aptests/test_attach_detach_multi_ue.py \
```

Comment:
We are adding the same test cases in 2 folders `magma/lte/gateway/python/integ_tests` and `magma/lte/gateway/python/integ_tests/federated_tests/s1aptests/`. Its almost impossible to always maintain same changes in both the places. I see the only difference between the test cases at the 2 places is:
```
<         self._s1ap_wrapper = s1ap_wrapper.TestWrapper(federated_mode=True)
---
>         self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
```

Is it not possible to use the same test cases for both the scenario? We may get this variable `federated_mode` by either passing one argument to pytest command or setting one ENVIRONMENT variable and getting it inside the test case to understand whether `federated_mode` is `true` or `false`. This will reduce the effort to maintain the new changes in both the places.

Reply:
@VinashakAnkitAman 
We are re-using test here for s6a over FEG but there will be new tests (or modification of some of these tests) for validation Gx, Gy, SGs, and 5G interfaces like nPCF etc. I also noted that current LTE integ tests does not cover test related to Gx, Gy, SGs etc.

Also when we run test from same folder need to make sure not all test are relevant/or run against Fed setup
e.g. `make fed_integ_test TESTS=s1aptests/test_x2_handover.py`  may not be relevant/supported against fed gateway tests.